TwilioVerifySDK/TwilioVerify/Sources/Data/Storage.swift

```diff
@@ -120,12 +125,36 @@ private extension Storage {
   func updateVersion(version: Int) {
     userDefaults.set(version, forKey: Constants.currentVersionKey)
   }
+  
+  func clearItemsWithoutService() throws {
+    let migration = AddKeychainServiceToFactors(secureStorage: secureStorage)
+    let migrationResult = migration.migrate(data: try secureStorage.getAll(withServiceName: Constants.service))
+    for result in migrationResult {
+      try removeValue(for: result.key)
+    }
+  }
+  
+  func previousClearStorageOnReinstallValue() -> Bool? {
+    guard let clearStorageOnReinstallValue = try? get(Constants.clearStorageOnReinstallKey) else {
+      return nil
+    }
+    return Bool(String(decoding: clearStorageOnReinstallValue, as: UTF8.self))
+  }
+  
+  func updateClearStorageOnReinstall(value: Bool) {
+    guard let clearStorageOnReinstallValue = value.description.data(using: .utf8) else {
+      return
+    }
+    try? secureStorage.save(clearStorageOnReinstallValue, withKey: Constants.clearStorageOnReinstallKey, withServiceName: nil)
```

Comment:
What do you think about adding a prefix `twilioVerify` for this `clearStorageOnReinstallKey`, just to ensure this value is only being used by the SDK?

Reply:
Or we can use the service we added for keychain items, wdyt?

Reply:
Yeah that's better 👍